frontend/app/src/i18n/en.json

```diff
@@ -316,17 +316,23 @@
+        "send_tokens_q": "Why am I charged a transaction fee when sending tokens?",
+        "send_tokens_a": "When sending tokens as a chat message, the transfer is performed by the ledger for the given token which incurs a mandatory fee. This fee is burned, slightly reducing the token supply. The fees for the various tokens we support are as follows: <ul><li>CHAT 0.001</li><li>ICP 0.0001</li><li>ckBTC 0.0000001</li><li>SNS1 0.00001</li></ul>",
+        "wallet_q": "How does my wallet work?",
+        "wallet_a": "When an OpenChat user is created, they automatically have an account for a bunch of tokens (ICP, CHAT, ckBTC, SNS1). This account will initially be empty but you can top up the account by transfering tokens to its address. Once you have some tokens in your OpenChat wallet you will be able to send them directly to the account of any other OpenChat user via a special kind of message. You can also use the ICP (soon CHAT) in your wallet to upgrade to Diamond membership. If you would like to withdraw the tokens you can do so by opening your wallet from the main menu and clicking the \"Manage\" button. This screen gives you the option to withdraw your tokens to any other address of your choosing.",
         "shortcuts_q": "Which shortcut commands are available?",
-        "shortcuts_a": "There are several shortcut commands that you can use from the message entry input box. <br /><br /> `/poll` - to create a new poll <br /> `/search xyz` - to search the current chat for the term \"xyz\" <br />`/gif kittens` - to search for kitten gifs <br />`/icp 0.1` - to send 0.1 ICP to someone <br />`/faq question_id*` - to create and post a formatted link to a specific FAQ question <br />`/diamond` - to create and post a formatted link with the details of diamond membership <br/> <br/>Commands which accept an argument can also be used without the argument.<br /><br />* Available faq question IDs are: ios_app, android_app, find_groups, style_messages, storage, airdrop, security, send_icp, icp_account, roadmap, shortcuts"
+        "shortcuts_a": "There are several shortcut commands that you can use from the message entry input box. <br /><br /> `/poll` - to create a new poll <br /> `/search xyz` - to search the current chat for the term \"xyz\" <br />`/gif kittens` - to search for kitten gifs <br />`/icp 0.1` - to send 0.1 ICP to someone <br />`/faq question_id*` - to create and post a formatted link to a specific FAQ question <br />`/diamond` - to create and post a formatted link with the details of Diamond membership <br/> <br/>Commands which accept an argument can also be used without the argument.<br /><br />* Available faq question IDs are: ios_app, android_app, find_groups, style_messages, storage, airdrop, security, send_icp, icp_account, roadmap, shortcuts",
```

Comment:
This answer references the faq shortcut `send_icp` but you have renamed it to `send_tokens` - so I think the description of the shortcut needs to be changed (and probably also need to double check that the shortcut still functions with the new name - hopefully it's not hardcoded elsewhere?)